rpc_util.go

```diff
@@ -297,21 +297,34 @@ func (o FailFastCallOption) before(c *callInfo) error {
 func (o FailFastCallOption) after(c *callInfo, attempt *csAttempt) {}
 
 // OnFinish returns a CallOption that configures a callback to be called when
-// the call completes.
+// the call completes. The error passed to the callback is the status of the
+// RPC, and may be nil. Must only be called once. This is mainly used in the
+// context of streaming interceptors, to be notified when the RPC completes
```

Comment:
Maybe avoid the word "context" since that has a specific meaning, and go with "This is mainly to be used by..." or "This is intended to be used by..."?

Reply:
Oh oof double meaning there "context of streaming interceptors"...switched to this is intended to be used by, since that is why I wrote it, for a streaming interceptor :).